code/modules/clothing/under/color.dm

```diff
@@ -48,124 +40,117 @@
 	desc = "It's standardised SCC prisoner-wear. Its suit sensors are stuck in the \"Fully On\" position.\nThis one has \"Cell 4\" marked on it."
 	id = "Cell 4"
 
-/obj/item/clothing/under/color/pink
-	name = "pink jumpsuit"
-	icon_state = "pink"
-	item_state = "p_suit"
-	worn_state = "pink"
+/obj/item/clothing/under/color/black
+	name = "black jumpsuit"
+	desc = "A bleak black jumpsuit"
+	icon_state = "black"
+	item_state = "black"
 
-/obj/item/clothing/under/color/red
-	name = "red jumpsuit"
-	icon_state = "red"
-	item_state = "r_suit"
-	worn_state = "red"
+/obj/item/clothing/under/color/grey
+	name = "grey jumpsuit"
+	desc = "A generic grey jumpsuit"
+	icon_state = "grey"
+	item_state = "grey"
 
 /obj/item/clothing/under/color/white
 	name = "white jumpsuit"
+	desc = "A wrinkly white jumpsuit"
 	icon_state = "white"
-	item_state = "w_suit"
-	worn_state = "white"
+	item_state = "white"
 
-/obj/item/clothing/under/color/yellow
-	name = "yellow jumpsuit"
-	icon_state = "yellow"
-	item_state = "y_suit"
-	worn_state = "yellow"
-
-/obj/item/clothing/under/psyche
-	name = "psychedelic jumpsuit"
-	desc = "Groovy!"
-	icon_state = "psyche"
-	item_state = "psyche"
-	worn_state = "psyche"
-
-/obj/item/clothing/under/rainbow
-	name = "rainbow"
-	desc = "rainbow"
-	icon_state = "rainbow"
-	item_state = "rainbow"
-	worn_state = "rainbow"
-
-/obj/item/clothing/under/lightblue
-	name = "lightblue jumpsuit"
-	desc = "lightblue"
-	icon_state = "lightblue"
-	item_state = "b_suit"
-	worn_state = "lightblue"
-
-/obj/item/clothing/under/aqua
-	name = "aqua jumpsuit"
-	desc = "aqua"
-	icon_state = "aqua"
-	item_state = "b_suit"
-	worn_state = "aqua"
-
-/obj/item/clothing/under/purple
-	name = "purple jumpsuit"
-	desc = "purple"
-	icon_state = "purple"
-	item_state = "p_suit"
-	worn_state = "purple"
+/obj/item/clothing/under/color/darkred
+	name = "darkred jumpsuit"
+	desc = "A dark red jumpsuit"
+	icon_state = "darkred"
+	item_state = "darkred"
 
-/obj/item/clothing/under/lightpurple
-	name = "lightpurple jumpsuit"
-	desc = "lightpurple"
-	icon_state = "lightpurple"
-	item_state = "p_suit"
-	worn_state = "lightpurple"
+/obj/item/clothing/under/color/red
+	name = "red jumpsuit"
+	desc = "A reliable red jumpsuit"
```

Comment:
punctuation on all of these entries.
also, personally, i would prefer if the descriptions were neutral instead of telling you what to feel about the jumpsuit variants.

Reply:
Mm! Fair. I was very sleep deprived when doing the final pass on the ``color.dm`` organization and decided to try to be funny. I'll remove the alliterative descriptions to just their base name color

Reply:
Done!